src/components/form-elements/Input.stories.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import Input from "./Input";
+
+export default {
+  title: "Components/Form/Input",
+  parameters: {
+    component: Input
+  }
+};
+
+const useInput = args => {
+  const [value, setValue] = useState(args.value);
+
+  const onChange = event => {
+    setValue(event.target.value);
+  };
+
+  return { value, onChange };
+};
+
+export const Default = args => {
+  const props = useInput(args);
```

Comment:
I am not sure why we need this hook.

Reply:
I thought we want to also edit this input from the input itself